src/System.Private.Xml/tests/XmlTextReader/System.Xml.XmlTextReader.Tests/XmlTextReaderTestHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+
+namespace System.Xml.Tests
+{
+    internal static class XmlTextReaderTestHelper
+    {
+        internal static XmlTextReader CreateReader(string input)
+        {
+            var textReader = new XmlTextReader(input, XmlNodeType.Element, null);
+            return textReader;
+        }
+
+        internal static XmlTextReader CreateReader(string input, XmlNameTable nameTable)
```

Comment:
I haven't seen where the nameTable was really used in the testcase. possibly we could remove it?